console/src/main/webapp/manager/app/components/logger/logger.tpl.html

```diff
@@ -0,0 +1,170 @@
+            <tr>
+            <th class="header-filter">
+                <date model="logger.date"></date>
+            </th>
+            <th class="header-filter">
+                <select class="pull-right form-control input-sm" ng-model="logger.admin"
+                ng-options="t.key as t.value for t in logger.senders" choosen>
+                </select>
+                <span translate>logs.sender</span>
+            </th>
+            <th class="header-filter">
+                <select class="pull-right form-control input-sm" ng-model="logger.target"
+                ng-options="t.key as t.value for t in logger.targets" chosen>
+                </select>
+                <span translate>logs.target</span>
+            </th>
+            <th class="header-filter">
+                <select class="pull-right form-control input-sm" ng-model="logger.type" 
+                ng-options="t.key as t.value for t in logger.types" chosen>
+                </select>
+                <span translate>logs.type</span>
+            </th>
+            </tr>
+        </thead>
+        <thead ng-if="!logger.filter">
+            <tr>
+                <th translate>logs.date</th>
+                <th translate>logs.sender</th>
+                <th translate>logs.target</th>
+                <th translate>logs.type</th>
+            </tr>
+        </thead>
+    <tbody>
+        <tr dir-paginate="log in logger.logs | logs: logger.type:logger.admin:logger.target:logger.date | orderBy: '-date' | itemsPerPage: logger.itemsPerPage">
+            <td>{{::log.date | logDate}}
+            <td><a ng-link="user({id: log.admin, tab: 'infos'})" title="">{{::log.admin}}</a>
+            <!--Target-->
+            <td>
+            <!--user or user's role modification-->
+            <a ng-if="logger.getType(log) === 'USER'" ng-link="user({id: log.target, tab: 'infos'})" title="">{{::log.target}}</a>
+            <!--org-->
+            <a ng-if="logger.getType(log) === 'ORG'" ng-link="org({org: log.target, tab: 'infos'})" title="">
+                <span ng-if="logger.orgsId[log.target]">{{::logger.orgsId[log.target]}}</span>
+                <span ng-if="!logger.orgsId[log.target]">{{::log.target}}</span>
+            </a>
+            <!--role-->
+            <a ng-if="logger.getType(log) === 'ROLE'" ng-link="role({role: log.target, tab: 'infos'})" title="">
+                {{::log.target}}</a>
+            <!--refused or deleted-->
+            <span ng-if="logger.getType(log) === ''">{{::log.target}}</span>
+            </td>
+            <!--Type-->
+            <td ng-if="!log.changed"> {{::log.type}}
+            <td>
+            <a ng-if="log.changed && log.changed.field != log.target" title="" ng-click="logger.openLog(log)" href="javascript:void(0);">
+                {{::log.type + (log.changed.field ? ' (' + log.changed.field + ')' : '')}}
+            </a>
+            <span ng-if="log.changed && log.changed.field === log.target">
```

Comment:
In which case `changed.field === target`? I must be missing something here (-:

Reply:
Indeed... don't remember this specific case. I thing it was usefull before lasts code modifications... (=> deadcode now ?).

Reply:
Removed this useless case.